plugins/user_quota/server/quota.py

```diff
@@ -174,63 +174,55 @@ def _validatePolicy(self, policy):
                            dictionary as if the JSON was already decoded.
         :returns: a validate policy dictionary.
         """
-        validKeys = []
-        for key in dir(self):
-            if key.startswith('_validate_'):
-                validKeys.append(key.split('_validate_', 1)[1])
-        for key in list(policy):
-            if key.startswith('_'):
-                del policy[key]
+        validKeys = [k[10:] for k in dir(self) if k.startswith('_validate_')]
+        policy = {k: v for k, v in six.iteritems(policy) if not k.startswith('_')}
```

Comment:
A comment here explaining why we remove policy keys prefixed with `'_'` would be helpful.

Reply:
@manthey , any idea why this was happening?